<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HubWindow</class>
 <widget class="QMainWindow" name="HubWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HubWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="metoriteButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>570</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Meteorite</string>
    </property>
   </widget>
   <widget class="QPushButton" name="volcanoButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>570</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Volcano</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mammalButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>570</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Mammals</string>
    </property>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>600</width>
      <height>500</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="physicsButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>610</y>
      <width>391</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Toggle Physics</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
